array_api_compat/dask/array/_aliases.py

```diff
@@ -40,19 +39,25 @@
 isdtype = get_xp(np)(_aliases.isdtype)
 unstack = get_xp(da)(_aliases.unstack)
 
+# da.astype doesn't respect copy=True
 def astype(
     x: Array,
     dtype: Dtype,
     /,
     *,
     copy: bool = True,
-    device: Device | None = None
+    device: Optional[Device] = None
```

Comment:
this change only matters if someone runs mypy/pyright in python 3.9. Runtime is unaffected due to `from __future__ import annotations`.